.github/messages.yaml

```diff
@@ -0,0 +1,45 @@
+stale-issue: |
+  Hi there, thank you for your contribution!
+
+  This issue has been **automatically marked as stale** because it has not had recent activity. It will be closed automatically if no further activity occurs.
+
+  If you would like this issue to remain open please:
+
+    1. Verify that you can still reproduce the issue at hand
+    2. Comment that the issue is still reproducible and include:
+      - What OS and version you reproduced the issue on
+      - What steps you followed to reproduce the issue
+
+  **NOTE:** If this issue was closed prematurely, please leave a comment.
+
+  Thanks!
+close-issue: |
+  Hi again!
+
+  This issue has been closed since it has not had recent activity.
+
+  **NOTE:** If this issue was closed prematurely, please leave a comment.
+
+  Thanks!
+
+stale-pr: |
+  Hi there, thank you for your contribution!
+
+  This pull request has been **automatically marked as stale** because it has not had recent activity. It will be closed automatically if no further activity occurs.
+
+  If you would like this pull request to remain open please:
+
+    1. Rebase and verify the changes still work
+    2. Leave a comment with the current status
+
+  **NOTE:** If this pull request was closed prematurely, please leave a comment.
+
+  Thanks!
+close-pr: |
+  Hi again!
+
+  This pull request has been closed since it has not had recent activity.
+
+  **NOTE:** If this pull request was closed prematurely, please leave a comment.
+
+  Thanks!
```

Comment:
@jezdez followup (1/2) from https://github.com/conda/conda/pull/11249, here's the universal stale & close blurbs, let me know what you think!